frontend/pages/index.tsx

```diff
@@ -20,10 +32,15 @@ const messages = defineMessages({
 export default function Home(): React.ReactNode {
   const router = useRouter();
   const { formatMessage } = useIntl();
+  const f = (id) => formatMessage({ id });
   const { locale, locales, defaultLocale } = router;
 
   return (
-    <>
+    <div className={styles.container}>
+      <Head>
+        <title>Homepage</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
```

Comment:
If I remember correctly, we already provide the Head attribute in the PageWrapper component in `_app.tsx` so we can remove it from here, as well as the surrounding container since the `<main>` tag already takes care of wrapping.